js/private/js_devserver.bzl

```diff
@@ -0,0 +1,91 @@
+"Implementation details for js_devserver rule"
```

Comment:
I guess you're still working on this - rename to js_run_devserver everywhere?

Reply:
yup. about to push that change